
#x - The x coordinate of the lower left corner of a rectangle
#y - The y coordinate of the lower left corner of a rectangle
#width - The width of the rectangle
#height - The height of the rectangle
#trans_vel - The translational velocity of the robot
#rot_vel - The rotational velocity of the robot
#frequency - The frequency of the control loop
#wheel_width: The width of the wheels
#wheel_radius: The radius of the wheels
#wheel_base: The distance between the centers wheels
#waypoints_x: x coordinates of waypointss
#waypoints_y: y coordinates of waypointss

x: 3
y: 2
width: 4
height: 5
trans_vel: 2
rot_vel: 1

#waypoints_x: [3.0, 4.0, 5.0, 3.5, 2.0]
#waypoints_y: [3.0, 3.0, 4.0, 5.0, 4.0]
waypoints_x: [1.0, 2.0, 3.0, 4.0, 5.0]
waypoints_y: [0.0, 0.0, 0.0, 0.0, 0.0]


wheel_radius: 0.2
wheel_base: 1.2
frequency: 20
